.gitmodules

```diff
@@ -19,7 +19,7 @@
 	url = https://github.com/mphowardlab/neighbor.git
 [submodule "hoomd/extern/hipper"]
 	path = hoomd/extern/hipper
-	url = https://github.com/mphowardlab/hipper.git
+	url = https://github.com/glotzerlab/hipper.git
```

Comment:
@joaander I merged in your PR if you want to keep using my version. If not, we will either need to sync your fork or change back the URL once I make the bigger revisions to hipper. That's on my list to do either later this semester or early summer so that MPCD will support HIP builds

Reply:
Thanks, I'll sync the fork.

I really need to port the rest of HOOMD over to hipper to avoid further reliance on our really old fork of HIP.

Reply:
Yeah, it may be a little labor intensive but having to rely on compiled HIP libraries is frustrating when most users are building CUDA... I'll see how quickly I can get to updating hipper, and I'll let you know what happens when I move MPCD.

I noticed that in rocm 6, the HIP API is much closer to CUDA (e.g., `threadIdx_x` -> `threadIdx.x`, chevrons for kernel launch) so I think the porting might actually be slightly easier to do by global find and replace than it was when we first thought about it.

Reply:
OK, so I should wait for you to update hipper before I start working on porting `hoomd`, `md`, and `hpmc` to it?

Reply:
Yes, I think that will be the best use of your time. I may be able to help you with it too when we get there!

Reply:
That would be great!